plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -465,6 +485,75 @@ public void testDeletingEntireFileWithNonTupleDomainConstraint()
         assertThat(this.loadTable(tableName).newScan().planFiles()).hasSize(1);
     }
 
+    @Test
+    public void testMultiplePartitionedWithAggregationPushdown()
+    {
+        Session clientSession = sessionWithAggregationPushdown();
+
+        String tableName = format("test_%s_with_aggregation_pushdown", "multiple_partition");
+        assertUpdate(format("CREATE TABLE %s (userid int, country varchar, event_date date, state varchar) %s",
```

Comment:
TestIcebergV2 is a class focused on Iceberg V2 spec aspects.
General Iceberg connector functions are covered in `BaseIcebergConnectorTest`.
please move the test there.

Reply:
will remove from the V2